### PR TITLE
Update to new ppafm

### LINKED ISSUE #14: All tests involving relaxed scanning fail
As PPAFM has seen major changes, now some of the features of PPSTM dependent on PPAFM have been broken as mentioned in #12. For example running `tests/4N-coronene/run_test.sh` fails as the LJ field used to be generated with `python PATH_TO_YOUR_PROBE_PARTICLE_MODEL/generateLJFF.py -i YOUR_INPUT_FILE.cube --npy` but the `--npy` option has been replaced with `-f npy`. Example error attached.

image

This requires at least changing the way PPAFM is called within PPSTM.

### PR DESCRIPTION
**Pull request by LauriKurki**
Resolves #14.

Updated the scripts in the repository to use the new ppafm.io module. Also updated the tests to use the new ppafm cli-commands, e.g. `ppafm-generate-ljff`. Tests for FePc_Au still fail as `ppafm-relaxed-scan --pos -f npy --vib 3` causes a seg fault but otherwise all tests are working.

We should also update the documentation for creating the force fields and relaxed probe particle paths. Now running PPSTM requires PPAFM(=<0.2.0a3) to be installed with `pip install ppafm=0.2.0a3`. Force fields are generated with e.g.
```
ppafm-generate-ljff -i input.cube -f npy
ppafm-generate-elff -i input.cube -f npy
ppafm-relaxed-scan --pos -f npy
```

### CODE REVIEW

File: PPSTM_simple.py [hunk 1]
```diff
@@ -143,7 +143,7 @@
     print("Importing positions of PP from the PP-AFM calculations. Path for the data:")
     path_pos="Q%1.2fK%1.2f/" %(Q,K)
     print(path_pos)
-    tip_r, lvec, nDim = GU.load_vec_field( path_pos+'PPpos' ,data_format=data_format)
+    tip_r, lvec, nDim, atomic_info, atomic_info_or_head = io.load_vec_field( path_pos+'PPpos' ,data_format=data_format)
```
Comment (ondrejkrejci) [inline PPSTM_simple.py:146]: Wait here you have there twice: `atomic_info`  as you have `atomic_info_or_head`: it should be:
`tip_r, lvec, nDim, atomic_info_or_head = io. ...`

**Review by ondrejkrejci — CHANGES_REQUESTED** [review]
One mistake found.
Please also change: https://github.com/Probe-Particle/PPSTM/wiki#relaxed-scan part of the wiki with the newest way of installation. I have feeling that at least for linux g++ is not required anymore ... Thank you!

File: PPSTM_simple.py (hunk 1 above)
Comment (LauriKurki) [reply PPSTM_simple.py]: Good catch! Fixed in new commit

**Comment by LauriKurki** [comment]
Wiki updated at https://github.com/Probe-Particle/PPSTM/wiki#relaxed-scan. 

This page also needs updates https://github.com/Probe-Particle/PPSTM/wiki/Prepare-for-the-flexible-tip-simulations and it still refers to the nanosurf pages for creating the probe positions which we probably don't want?

**Comment by LauriKurki** [comment]
Yes, the only thing not working at the moment is the FePc test for two reasons.
1. *.in is not currently supported as an input, but should be soon (https://github.com/Probe-Particle/ppafm/issues/177 and https://github.com/Probe-Particle/ppafm/issues/39)

2. `ppafm-relaxed-scan --pos -f npy --vib 3` causes a seg fault in PPAFM

**Comment by ondrejkrejci** [comment]
Yes, I know that one, I have tried to looked into it, but it is pretty problematic in C++ and the question is if there is a market for it ....

**Pull request merged**